perses/tests/test_repex.py

```diff
@@ -188,5 +189,229 @@ def test_RESTCapableHybridTopologyFactory_repex_charge_mutation():
                        + data['ala-arg']['error'] ** 2
                        + data['lys-ala']['error'] ** 2
                        + data['ala-lys']['error'] ** 2)
+        assert DDG < 6 * dDDG, f"DDG ({DDG}) is greater than 6 * dDDG ({6 * dDDG})"
+
+
+@pytest.mark.gpu_needed
+def test_RESTCapableHybridTopologyFactory_repex_neutral_transformation():
+    """
+    Run CCC->CCCC and CCCC->CCC repex with the RESTCapableHybridTopologyFactory and make sure that the free energies are
```

Comment:
If we are going to merge this with phenol and paracetamol, we'll want to update the doc string here.